tools/VersionController/Models/VersionBumper.cs

```diff
@@ -296,7 +297,7 @@ private void UpdateDependentModules()
                                                    .Where(f => !f.Contains("Netcore") &&
                                                                !f.Contains("bin") &&
                                                                !f.Contains("dll-Help") &&
-                                                               !f.Contains("Stack"))
```

Comment:
`(!f.Contains("Stack") || WhiteList.Contains(f)` can be more readable. For example: `!ModuleFilter.IsAzureStackModule(f)`